src/crush/CrushWrapper.h

```diff
@@ -59,9 +59,9 @@ class CrushWrapper {
     DEFAULT_CHOOSE_ARGS = -1
   };
 
-  std::map<int32_t, std::string> type_map; /* bucket/device type names */
-  std::map<int32_t, std::string> name_map; /* bucket/device names */
-  std::map<int32_t, std::string> rule_name_map;
+  std::map<int32_t, std::string> type_map; /* bucket/device {type, name} e.g. {{0, "osd"}, {11, "root"}} */
```

Comment:
```suggestion
  std::map<int32_t, std::string> type_map; // id => bucket/device type name
```

"bucket/device type name" is a compound word in this context. and i'd suggest use `//` for starting a comment.